packages/commonwealth/client/scripts/views/components/sidebar/external_links_module.tsx

```diff
@@ -1,15 +1,30 @@
-import React from 'react';
-
 import 'components/sidebar/external_links_module.scss';
-
+import { categorizeSocialLinks } from 'helpers/link';
+import React from 'react';
 import app from 'state';
+import { useGetCommunityByIdQuery } from 'state/api/communities';
 import { CWIcon } from '../component_kit/cw_icons/cw_icon';
+import CWCircleMultiplySpinner from '../component_kit/new_designs/CWCircleMultiplySpinner';
 
 export const ExternalLinksModule = () => {
-  if (!app.chain) return;
-  const meta = app.chain.meta;
-  const { remainingLinks, discords, elements, telegrams, githubs, twitters } =
-    meta.categorizeSocialLinks();
+  const { data: community, isLoading } = useGetCommunityByIdQuery({
+    id: app.activeChainId() || '',
+    enabled: !!app.activeChainId(),
+  });
+
+  if (!app.chain || !community) return;
+
+  if (isLoading) return <CWCircleMultiplySpinner />;
+
+  const {
+    discords,
+    elements,
+    githubs,
+    remainingLinks,
+    // slacks, TODO: maybe add this here as well -- ask product?
```

Comment:
ask and implement or remove comment

Reply:
per product: we should display it like a normal link for now